test/e2e/scenarios/start/benefitType.js

```diff
@@ -25,3 +28,23 @@ Scenario('When I enter a non PIP benefit type, I am taken to the download form p
     I.seeInCurrentUrl(paths.identity.downloadAppointeeForm);
 
 });
+
+Scenario('When I enter different non PIP benefit types, I am taken to the download form page', (I) => {
+    benefitTypes.forEach((benefitType) => {
+    if(benefitType != 'Personal Independence Payment (PIP)'){
```

Comment:
I guess this needs tabulator. 
And this is safer in the if statement (!==) rather than what you currently have. 

Apart from that, this piece of code looks better. As an alternative you can simplify the if in one line using this: 
` benefitTypes.filter(benefitType => benefitType !== 'Personal Independence Payment (PIP)').forEach(benefitType => {
        I.enterBenefitTypeAndContinue(benefitType);
        I.see(benefitType);
        I.seeInCurrentUrl(paths.identity.downloadAppointeeForm);
        I.see('Continue to form download');
        I.click('Back'); //click on Back link
    });`

Reply:
If there are any changes require please feel free to add and there are different ways we can write. If you feel that if there is any change is required please feel free to add it.